### PR TITLE
Fallback to thumbnail #83

### PR DESCRIPTION
**Pull request by Gozala**
# Work in progress

### CODE REVIEW

File: node_modules/browser/page-switch.js [hunk 1]
```diff
@@ -12,17 +12,21 @@ const {Deck} = require('./deck');
 const {select, remove} = require('./deck/actions');
 
 const readThumbnailURI = uri =>
-  'none' && `url(https://raw.githubusercontent.com/gordonbrander/website-tiles/master/${url.getDomainName(uri)}.png)`;
+  'none' && `https://raw.githubusercontent.com/gordonbrander/website-tiles/master/${url.getDomainName(uri)}.png`;
```
Comment (paulrouget) [inline node_modules/browser/page-switch.js]: `none` doesn't make sense anymore.

File: node_modules/browser/page-switch.js [hunk 2]
```diff
@@ -12,17 +12,21 @@ const {Deck} = require('./deck');
...
-  DOM.div({
+  DOM.img({
     key: `tab-${item.get('id')}`,
     className: 'tab',
-    style: {
-      backgroundImage: readThumbnailURI(item.get('location'))
-    },
+    src: readThumbnailURI(item.get('location')),
+    onError: item.get('thumbnail') && (event => {
+      event.target.src = URL.createObjectURL(item.get('thumbnail'));
```
Comment (paulrouget) [inline node_modules/browser/page-switch.js]: Don't forget to revoke. And can't the URL be created earlier? Right after retrieving the blob.

File: node_modules/browser/web-viewer.js [hunk 3]
```diff
@@ -67,6 +67,15 @@ const IFrame = Element('iframe', {
       }
     }
 
+    if (current != past && current == 'loaded') {
+      if (node.getScreenshot) {
+        setTimeout(() => {
+          var request = node.getScreenshot(100, 62.5, 'image/jpeg');
```
Comment (paulrouget) [inline node_modules/browser/web-viewer.js]: Use `devicePixelRatio` for the dimensions.

**Comment by Gozala** [comment]
@paul this was not meant to be reviewed yet that's why title said "Work in progress" ;)

**Comment by Gozala** [comment]
Close in favour of #220

**Pull request closed without merging**